src/styles/main.scss

```diff
@@ -1,3 +1,69 @@
-body {
+$size: 100px;
```

Comment:
It is better to give more meaningful names so that it is immediately clear what this variable is responsible for (what size is this exactly?).

Also, all variables can be exported to a separate file.